### PR TITLE
Make "sitemap" & "archive" a separate add-on

### PR DESCRIPTION
**Pull request by skiane**
Benefits:
- snews core file is smaller
- easier to suppress sitemap or archive page (simply remove the add-on file)

Cons ?

Please, review and comment.

### CODE REVIEW

File: snews.php [hunk 1]
```diff
@@ -2188,6 +2051,8 @@ function center() {
 // ROUTES
 addRoute('login', l('login'), 'login');
 addRoute('contact', l('contact'), 'contact');
-addRoute('sitemap', l('sitemap'), 'sitemap');
-addRoute('archive', l('archive'), 'archive');
+
+// LOAD ADDONS
+readAddons();
```
Comment (snewscms) [inline snews.php:2056]: This function won't work if call addon "hello".
We must kepp call function before function l($var)

**Comment by snewscms** [comment]
Yes I agree, sometime ago I think to change this functions as addon

File: snews.php (hunk 1 above)
Comment (skiane) [inline snews.php:2056]: I see but then I have a problem with load_lang() in sitemap.
I have to figure out what the sequence should be.

**Comment by snewscms** [comment]
Updat my last version, I think is working fine.
I made sitemap, contact and archive as addon (not auto-load). I think is best only load addon if you request it, less code, less memory.
Please check

**Pull request closed without merging**